alacritty/res/text.v.glsl

```diff
@@ -39,8 +45,15 @@ void main()
     vec2 cellPosition = cellDim * gridCoords;
 
     if (backgroundPass != 0) {
-        vec2 finalPosition = cellPosition + cellDim * position;
-        gl_Position = vec4(projectionOffset + projectionScale * finalPosition, 0.0, 1.0);
+        vec2 backgroundDim = cellDim;
+        // We use 1 to identify wide chars.
```

Comment:
```suggestion
```

The constant tells us why, the `1` this comment talks about is nowhere to be found.